add html quiz questions

terrifricker opened the pull request.

DavidMatthewFraser commented:
Hi Terri,

Thanks for working on this quiz questions!

There is a problem with the last question:
`answers: ["<header>, <body>, <head>, <nav>"]` should be `answers: ["<header>", "<body>", "<head>", "<nav>"]`

And the second to last question:
`answers: ["<html>, <body>, <head>", "<body>, <html>, <head>",
              "<head>, <body> <html>", "<html>, <head>, <body>"],`
As our quiz.js doesn't like more than one html tag in a question. 
I think this has to do with `answer()` on lines 28-41 of `quiz.js`

I wonder what SyntappZ's suggestions are on this second problem.

terrifricker commented:
I added a commit to this branch to update the syntax on the <footer> question.  I know how to resolve the second problem of change all the '<' and '>' tags, but my local repository was getting error codes because something (maybe a commit) had failed.  SO, I deleted my remote repository and forked again from your original and deleted my local copy of the repository.

I'll start over, but I think I'll have to make a new branch and pull request.

terrifricker commented:
See pull request #82 'update quiz javascript' for my idea for a fix to the answer function starting on line 28.
